TypeError: Cannot read property 'tags' of undefined

Error output when I use `cypress-tags run -e TAGS='not @Ignore' --env mode=aws`

```bash
> cypress-tags run -e TAGS='not @Ignore' --env mode=aws

/opt/jenkins/workspace/<project-name>/node_modules/cypress-cucumber-preprocessor/cypress-tags.js:32
  const featureTags = parsedFeature.feature.tags;
                                            ^

TypeError: Cannot read property 'tags' of undefined
    at paths.forEach.featurePath (/opt/jenkins/workspace/<project-name>/node_modules/cypress-cucumber-preprocessor/cypress-tags.js:32:45)
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/opt/jenkins/workspace/<project-name>/node_modules/cypress-cucumber-preprocessor/cypress-tags.js:28:7)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:877:12)
    at internal/main/run_main_module.js:21:11
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! testCi: `cypress-tags run -e TAGS='not @Ignore' --env mode=aws`
npm ERR! Exit status 1
npm ERR! 
```

If I use `cypress run -e TAGS='not @Ignore' --env mode=aws` then I see no issue but it doesn't process TAGS attribute

node version: 11.13.0

From package.json
```json
"devDependencies": {
    "axe-core": "^3.3.1",
    "chai": "^4.2.0",
    "cypress": "^3.4.1",
    "cypress-axe": "^0.5.1",
    "cypress-cucumber-preprocessor": "^1.16.0"
  },
```

--- a/cypress-tags.js
+++ b/cypress-tags.js
@@ -38,6 +38,11 @@ paths.forEach(featurePath => {
   const spec = `${fs.readFileSync(featurePath)}`;
   const parsedFeature = new Parser().parse(spec);
 
+  if (!parsedFeature.feature) {
+    debug(`Feature: ${featurePath} is empty`);
+    return;
+  }
+
   const featureTags = parsedFeature.feature.tags;
   const featureShouldRun = shouldProceedCurrentStep(featureTags, envTags);
   const taggedScenarioShouldRun = parsedFeature.feature.children.some(

